nft_ingester/src/api/dapi/rpc_asset_convertors.rs

```diff
@@ -234,16 +235,29 @@ fn extract_collection_metadata(
         meta.set_item("name", name.clone());
     }
 
-    let symbol = safe_select(chain_data_selector, "$.symbol");
-    if let Some(symbol) = symbol {
-        meta.set_item("symbol", symbol.clone());
+    let link_fields = vec!["name", "symbol"];
+    for name in link_fields {
+        let value = safe_select(chain_data_selector, format!("$.{}", name).as_str());
+        match value {
+            Some(symbol) => {
+                meta.set_item(name, symbol.clone());
+            },
+            None => {
+                meta.set_item(name, empty_string_value.clone());
```

Comment:
`"".into()`  
Won't do the trick? I don't thing we need to create an empty string as a dedicated variable.

Reply:
Good trick)